readme.md

```diff
@@ -0,0 +1,144 @@
+<!--
+*** https://www.markdownguide.org/basic-syntax/
+-->
+# Label Buster
+
+The Label Buster is a product that allows you to better understand requirements for selling food in QLD.
+
+
+## Table of Contents
+
+- [Label Buster](#label-buster)
+  - [Table of Contents](#table-of-contents)
+  - [About The Project](#about-the-project)
+    - [Built With](#built-with)
+    - [Decisions](#decisions)
+      - [Form.io](#formio)
+      - [Squiz](#squiz)
+      - [Misc](#misc)
+      - [Development and build tools](#development-and-build-tools)
+        - [Eslint](#eslint)
+        - [Prettier](#prettier)
+        - [Karma](#karma)
+        - [Parcel](#parcel)
+    - [Unknowns](#unknowns)
+      - [Form.io](#formio-1)
+      - [Squiz](#squiz-1)
+    - [Resources](#resources)
+  - [Getting Started](#getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Installation](#installation)
+  - [Usage](#usage)
+    - [Setting up Squiz with Form.io](#setting-up-squiz-with-formio)
+    - [Leveraging Form.io](#leveraging-formio)
+  - [Decisions](#decisions-1)
+
+
+## About The Project
+
+This project is a single page application embedded within a Squiz Matrix page. It uses Form.io to manage both the rendering and back end of the application, and SWE components to achieve the design.
+
+### Built With
+* [Form.io](https://www.form.io/)
+* [Squiz Matrix](https://www.squiz.net/platform/products/cms)
+* [SWE](https://github.com/qld-gov-au/swe)
```

Comment:
Probably worth pointin to SWE4? https://qld-gov-au.github.io/web-template-release/

Reply:
Updated.